### ISSUE TITLE
Add Italian language

### ISSUE THREAD

**Issue by davidullo** [BlueWallet/BlueWallet#371]

**Comment by marcosrdz** [comment]
That would be greatly appreciated! :-)

**Comment by davidullo** [comment]
Just made a pull request, hit me up if I'm missing something else 👌

**Comment by ncoelho** [comment]
> Just made a pull request, hit me up if I'm missing something else 👌

Yes, can you take a look on this commit and see what is missing. Basically just two things, adding it to the options in the view and a label. Thank you :)

https://github.com/BlueWallet/BlueWallet/commit/f75c17950478aefd8ce17ed25fe033164b027418

**Comment by davidullo** [comment]
> > Just made a pull request, hit me up if I'm missing something else 👌
> 
> Yes, can you take a look on this commit and see what is missing. Basically just two things, adding it to the options in the view and a label. Thank you :)
> 
> [f75c179](https://github.com/BlueWallet/BlueWallet/commit/f75c17950478aefd8ce17ed25fe033164b027418)

I can't edit edit or propose changes for that file because I'm not in a "branch"

**Comment by ncoelho** [comment]
> > > Just made a pull request, hit me up if I'm missing something else 👌
> > 
> > 
> > Yes, can you take a look on this commit and see what is missing. Basically just two things, adding it to the options in the view and a label. Thank you :)
> > [f75c179](https://github.com/BlueWallet/BlueWallet/commit/f75c17950478aefd8ce17ed25fe033164b027418)
> 
> I can't edit edit or propose changes for that file because I'm not in a "branch"

You just need to fork it, create a new branch. Then do the changes locally, commit the changes and then submit a PR from that branch you created

**Comment by davidullo** [comment]
Thanks! I think I'm done 😃
